custom_components/adaptive_lighting/translations/zh-Hans.json

```diff
@@ -0,0 +1,309 @@
+          "name": "sleep_rgb_color"
+        },
+        "sleep_color_temp": {
+          "description": "睡眠模式中的色温（当`sleep_rgb_or_color_temp`为`color_temp`时使用），以开尔文为单位。😴",
+          "name": "sleep_color_temp"
+        },
+        "sunrise_offset": {
+          "description": "用正负偏移秒调整日出时间。⏰",
+          "name": "sunrise_offset"
+        },
+        "sunset_offset": {
+          "description": "用正负偏移秒调整日落时间。⏰",
+          "name": "sunset_offset"
+        },
+        "transition": {
+          "description": "灯光变化时的过渡持续时间，以秒为单位。🕑",
+          "name": "transition"
+        },
+        "transition_until_sleep": {
+          "description": "启用时，自适应照明将将睡眠设置视为最小值，在日落后过渡到这些值。🌙",
+          "name": "transition_until_sleep"
+        },
+        "adapt_only_on_bare_turn_on": {
+          "description": "当首次打开灯光时。如果设置为`true`，仅在没有指定颜色或亮度的情况下，AL才进行适应。❌🌈 例如，这可以防止在激活场景时进行适应。如果为`false`，则不考虑初始`service_data`中是否存在颜色或亮度，AL都会适应。需要启用`take_over_control`。🕵️",
+          "name": "adapt_only_on_bare_turn_on"
+        },
+        "adapt_delay": {
+          "description": "灯光打开和自适应照明应用更改之间的等待时间（秒）。可能有助于避免闪烁。⏲️",
+          "name": "adapt_delay"
+        },
+        "autoreset_control_seconds": {
+          "description": "在若干秒后自动重置手动控制。设置为0以禁用。⏲️",
+          "name": "autoreset_control_seconds"
+        },
+        "brightness_mode": {
+          "description": "要使用的亮度模式。可能的值为`default`、`linear`和`tanh`（使用`brightness_mode_time_dark`和`brightness_mode_time_light`）。📈",
+          "name": "brightness_mode"
+        },
+        "brightness_mode_time_dark": {
+          "description": "（如果`brightness_mode='default'`将被忽略）日出/日落之前/之后亮度逐渐增加/减少的持续时间，以秒为单位。📈📉",
+          "name": "brightness_mode_time_dark"
+        },
+        "brightness_mode_time_light": {
+          "description": "（如果`brightness_mode='default'`将被忽略）日出/日落之后/之前亮度逐渐增加/减少的持续时间，以秒为单位。📈📉。",
+          "name": "brightness_mode_time_light"
+        }
+      }
+    }
+  },
+  "sensor": {
+    "brightness": {
+      "description": "显示当前亮度。💡",
+      "name": "亮度"
+    },
+    "color_temp": {
+      "description": "显示当前颜色温度，以开尔文为单位。🔥",
+      "name": "颜色温度"
+    },
+    "color": {
+      "description": "显示当前颜色值（RGB格式）。🌈",
+      "name": "颜色"
+    },
+    "status": {
+      "description": "显示自适应照明的状态。🔄",
+      "name": "状态"
+    },
+    "mode": {
+      "description": "显示自适应照明的当前模式。🌞🌙",
+      "name": "模式"
+    },
+    "control": {
+      "description": "显示灯光是否处于手动控制模式。🔒",
+      "name": "控制"
+    }
+  },
+  "input": {
+    "name": {
+      "description": "为自适应照明实例选择一个名称",
+      "name": "名称"
+    },
+    "config_defaults": {
+      "description": "设置所有选项的默认值，以便在配置流程中使用。⚙️",
+      "name": "config_defaults"
+    },
+    "take_over_control": {
+      "description": "为选择的灯光实体启用手动控制时，是否自动接管控制权。",
+      "name": "take_over_control"
+    },
+    "keep_manual_control_on": {
+      "description": "重新应用自适应照明设置后，是否保持之前手动控制的状态。",
+      "name": "keep_manual_control_on"
```

Comment:
Thanks a lot for doing this! However, it looks like this PR includes strings that do not even exist?

Did you use an AI or something?